.travis.yml

```diff
@@ -26,7 +31,7 @@ before_install:
   #- sudo sh -c "echo \"deb http://packages.ros.org/ros-shadow-fixed/ubuntu $ROS_CI_DESKTOP main\" > /etc/apt/sources.list.d/ros-latest.list"
   - wget http://packages.ros.org/ros.key -O - | sudo apt-key add -
   # Update python 2.7 version
-  - sudo apt-add-repository ppa:fkrull/deadsnakes-python2.7 -y
+  - sudo apt-add-repository ppa:fkrull/deadsnakes -y
```

Comment:
ppa deprecatead : use https://launchpad.net/~deadsnakes/+archive/ubuntu/ppa
Why do you need it btw ?